From 99046f32ab1d758a4fbcb20b46e7c9eb9966c432 Mon Sep 17 00:00:00 2001
From: Praveen Diwakar <praveen.diwakar@intel.com>
Date: Mon, 23 Feb 2015 16:02:29 +0530
Subject: [PATCH 07/15] ASoc: Intel: Add new control to support FBA UL/DL lock
 implementation

Issue: GMINL-6292

This patch adds new control to support FBA UL/DL.
FBA ALGO lock will be used in the case of switch
between wideband-narrowband.

Change-Id: I99b884607f8861f486a7d4ad68618a4f7d1f7d5e
Signed-off-by: Praveen Diwakar <praveen.diwakar@intel.com>
Signed-off-by: Omair Mohammed Abdullah <omair.m.abdullah@intel.com>
Signed-off-by: Divya Prakash <divya1.prakash@intel.com>
Signed-off-by: Fabien Marotte <fabien.marotte@intel.com>
---
 sound/soc/intel/platform-libs/atom_controls.h | 1 +
 sound/soc/intel/platform-libs/atom_pipes.h    | 3 ++-
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/sound/soc/intel/platform-libs/atom_controls.h b/sound/soc/intel/platform-libs/atom_controls.h
index dc9419d..363b867 100644
--- a/sound/soc/intel/platform-libs/atom_controls.h
+++ b/sound/soc/intel/platform-libs/atom_controls.h
@@ -184,6 +184,7 @@ enum sst_cmd {
 	SBA_VB_SET_IIR          = 129,
 	SBA_SET_SSP_SLOT_MAP	= 130,
 	AWARE_ENV_CLASS_PARAMS	= 130,
+	FBA_VB_ALGO_LOCK	= 132,
 	VAD_ENV_CLASS_PARAMS	= 2049,
 };
 #endif /* __ATOM_CONTROLS_H__ */
diff --git a/sound/soc/intel/platform-libs/atom_pipes.h b/sound/soc/intel/platform-libs/atom_pipes.h
index 0b6ba92..183d3e5 100644
--- a/sound/soc/intel/platform-libs/atom_pipes.h
+++ b/sound/soc/intel/platform-libs/atom_pipes.h
@@ -299,7 +299,8 @@ enum sst_dfw_module_id {
 	SST_DFW_MODULE_ID_WNR		  = 0x009D,
 
 	SST_DFW_MODULE_ID_LOG		  = 0xFF00,
-
+	SST_DFW_MODULE_ID_VOICE_DL            = 0x00A0,
+	SST_DFW_MODULE_ID_VOICE_UL            = 0x00A1,
 	SST_DFW_MODULE_ID_TASK		  = 0xFFFF,
 };
 
-- 
1.9.1

